controls/V-13621.rb

```diff
@@ -119,7 +119,10 @@
       describe file(file) do
         its('owner') { should be_in authorized_sa_user_list }
         its('group') { should be_in authorized_sa_group_list }
-        its('mode') { should cmp '640' }
+        it { should_not be_executable }
```

Comment:
Please update these to use the new matcher I created. Be_less_permissive_than in the file resource.